indico/modules/events/timetable/client/js/EntryPopup.tsx

```diff
@@ -94,6 +94,40 @@ function TimetablePopupContent({
     dispatch(actions.setDraftEntry(draftEntry));
   };
 
+  const onDelete = async (e: React.MouseEvent<HTMLButtonElement>) => {
+    if (!draftEntry.id) {
+      return;
+    }
+
+    onClose();
+    e.preventDefault();
+    e.stopPropagation();
+
+    const deleteURL = {
+      [EntryType.Break]: breakURL({event_id: eventId, break_id: draftEntry.id}),
+      [EntryType.SessionBlock]: sessionBlockURL({
+        event_id: eventId,
+        session_block_id: draftEntry.id,
+      }),
+      [EntryType.Contribution]: contributionURL({event_id: eventId, contrib_id: draftEntry.id}),
+    }[draftEntry.type];
+
+    try {
+      const {data} = await indicoAxios.delete(deleteURL);
+      console.log(data);
+
+      if (draftEntry.type === EntryType.Break) {
+        dispatch(actions.deleteBreak(draftEntry.id));
+      } else if (draftEntry.type === EntryType.SessionBlock) {
+        dispatch(actions.deleteBlock(draftEntry.id));
+      } else if (draftEntry.type === EntryType.Contribution) {
+        dispatch(actions.unscheduleEntry(draftEntry));
+      }
```

Comment:
This would generally be fine, but as we currently do this slightly differently, for example in the `TimetableManagementModal.tsx` without if/else like this, maybe let's stick to that.

Reply:
Also, line 124, maybe inconsistent to delete in different ways